builtin/types/config/config.pb.go

```diff
@@ -92,35 +80,13 @@ func (m *ConfigInitRequest) GetSettings() []*UpdateSetting {
 }
 
 type GetSetting struct {
-	Key                  string   `protobuf:"bytes,1,opt,name=key,proto3" json:"key,omitempty"`
```

Comment:
you are on the wrong version of gogo protobuf so please don't check in changed pb.gos